Gemfile.lock

```diff
@@ -1,62 +1,60 @@
 GEM
   remote: https://rubygems.org/
   specs:
-    better_errors (2.1.1)
+    better_errors (2.5.0)
       coderay (>= 1.0.0)
-      erubis (>= 2.6.6)
+      erubi (>= 1.0.0)
       rack (>= 0.9.0)
-    byebug (9.0.6)
-    coderay (1.1.1)
-    commander (4.4.3)
-      highline (~> 1.7.2)
-    database_cleaner (1.6.1)
+    byebug (10.0.2)
+    coderay (1.1.2)
+    commander (4.4.7)
+      highline (~> 2.0.0)
+    database_cleaner (1.7.0)
     diff-lcs (1.3)
-    erubis (2.7.0)
-    event_sourcery (0.13.0)
-    event_sourcery-postgres (0.3.0)
-      event_sourcery (>= 0.10.0)
+    erubi (1.7.1)
+    event_sourcery (0.22.0)
+    event_sourcery-postgres (0.8.0)
+      event_sourcery (>= 0.14.0)
       pg
-      sequel (~> 4.38)
-    highline (1.7.8)
-    method_source (0.8.2)
-    mustermann (1.0.0)
+      sequel (>= 4.38)
+    highline (2.0.0)
+    method_source (0.9.2)
+    mustermann (1.0.3)
     pg (0.20.0)
-    pry (0.10.4)
+    pry (0.12.2)
       coderay (~> 1.1.0)
-      method_source (~> 0.8.1)
-      slop (~> 3.4)
-    pry-byebug (3.4.2)
-      byebug (~> 9.0)
+      method_source (~> 0.9.0)
+    pry-byebug (3.6.0)
+      byebug (~> 10.0)
       pry (~> 0.10)
-    rack (2.0.3)
-    rack-protection (2.0.0)
+    rack (2.0.6)
+    rack-protection (2.0.4)
       rack
-    rack-test (0.6.3)
-      rack (>= 1.0)
-    rake (12.0.0)
-    rspec (3.6.0)
-      rspec-core (~> 3.6.0)
-      rspec-expectations (~> 3.6.0)
-      rspec-mocks (~> 3.6.0)
-    rspec-core (3.6.0)
-      rspec-support (~> 3.6.0)
-    rspec-expectations (3.6.0)
+    rack-test (1.1.0)
+      rack (>= 1.0, < 3)
+    rake (12.3.1)
+    rspec (3.8.0)
+      rspec-core (~> 3.8.0)
+      rspec-expectations (~> 3.8.0)
+      rspec-mocks (~> 3.8.0)
+    rspec-core (3.8.0)
+      rspec-support (~> 3.8.0)
+    rspec-expectations (3.8.2)
       diff-lcs (>= 1.2.0, < 2.0)
-      rspec-support (~> 3.6.0)
-    rspec-mocks (3.6.0)
+      rspec-support (~> 3.8.0)
+    rspec-mocks (3.8.0)
       diff-lcs (>= 1.2.0, < 2.0)
-      rspec-support (~> 3.6.0)
-    rspec-support (3.6.0)
-    sequel (4.47.0)
+      rspec-support (~> 3.8.0)
+    rspec-support (3.8.0)
```

Comment:
### [rspec-support](https://github.com/rspec/rspec-support)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 3.6.0 → 3.8.0

[_~~change-log~~, ~~source-code~~_]





<details>
<summary>Commits</summary>

A change of **66** commits. See the full changes on [the compare page](https://github.com/rspec/rspec-support/compare/v3.6.0...v3.8.0).

These are the first 10 commits:
- (f0af3fd) [Bump version to 3.7.0.pre](https://github.com/rspec/rspec-support/commit/f0af3fd74a94ff7bb700f6ba06dbdc67bba17fbf)
- (be67d3f) [Fixing strings if they're frozen literals.](https://github.com/rspec/rspec-support/commit/be67d3f083af7528a9cf6ab427c8ed02be727d70)
- (a9a4b09) [Merge pull request #316 from pat/string-literals](https://github.com/rspec/rspec-support/commit/a9a4b09e341611107d2c87a5fa7d53c5a6155e71)
- (738b3c3) [Explicitly get a module name using `name`.](https://github.com/rspec/rspec-support/commit/738b3c3d9dd467abc22817a69c4a530c5ad84c35)
- (b0b5795) [Merge pull request #317 from rspec/myron/fix-mocks-1161](https://github.com/rspec/rspec-support/commit/b0b5795757cad1cad435c268d2ed8837255bef5e)
- (d628be5) [Updates for frozen-string-literal compatibility.](https://github.com/rspec/rspec-support/commit/d628be587d1082c511efc2417345e8365209de65)
- (abc4124) [Merge pull request #320 from pat/frozen-string-literals](https://github.com/rspec/rspec-support/commit/abc412493345b247396ceddd3b9a6a3c7b507361)
- (5a04bb1) [object_formatter_spec.rb: fix tests on ia64](https://github.com/rspec/rspec-support/commit/5a04bb1c8321f147538dd8c8f720898bb22155ed)
- (d7d8053) [Merge pull request #321 from trofi/master](https://github.com/rspec/rspec-support/commit/d7d80534a9badb58f58949fe3355693e798f9f6c)
- (491e723) [Update changelog.](https://github.com/rspec/rspec-support/commit/491e7236856d93103718a8e30e8b9a415fbad28b)


</details>